lib/ansible/modules/cloud/vmware/vmware_guest_network.py

```diff
@@ -300,12 +314,14 @@ def get_network_info(self, vm_obj):
         nic_index = 0
         for nic in vm_obj.config.hardware.device:
             nic_type = None
+            directpath_io = 'N/A'
             if isinstance(nic, vim.vm.device.VirtualPCNet32):
                 nic_type = 'PCNet32'
             elif isinstance(nic, vim.vm.device.VirtualVmxnet2):
                 nic_type = 'VMXNET2'
             elif isinstance(nic, vim.vm.device.VirtualVmxnet3):
                 nic_type = 'VMXNET3'
+                directpath_io = True
```

Comment:
I think this value can be get from "nic.uptCompatibilityEnabled", not just set to "True".

Reply:
ah right!